tt_metal/hw/firmware/src/idle_erisc.cc

```diff
@@ -24,6 +24,7 @@
 #include "dataflow_api.h"
 #include "tt_metal/impl/dispatch/dispatch_address_map.hpp"
 
+#include "debug/watcher_common.h"
```

Comment:
why do you need this include?

Reply:
This header contains `clear_previous_launch_message_entry_for_watcher()` which is a function thats used in the main event loops when watcher is enabled. This makes workers clear stale launch message state so that watcher doesn't report fake kernels being run.